### PR TITLE
updated the README.md file

### PR DESCRIPTION
**Pull request by avia0**
# PR Details

## Checklist

- [x] Changes follows the best practices from the given [docs](https://docs.google.com/document/d/1xfYfruru0RFWOH23GRrRzosSlepJUTcakLFHLHdpEtc/edit?usp=sharing).
- [x] Updated CHANGELOG.md.
- [x] Ran unit tests locally.
- [x] Updated screenshots/sample logs if any.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,97 @@
...
-   
-   * Here's the [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md) guidelines to follow.
+We provide only actionable alerts, actively guiding you to block the attack source with the help of your existing defense systems. From the Nucleon dashboard, you can access threat intelligence assessments, threat alerts, insights and remediation guidance that contribute to a strengthened security posture.
 
-# Documentation
+The process is described in detail. For any questions or concerns please contact us through
+your Slack channel or at Support@nucleon.sh
 
-   * To know more about existing apps and integrations follow docs [here](https://help.sumologic.com/07Sumo-Logic-Apps).
+# Installation
```
Comment (himanshu219) [inline README.md:29]: Instead of Installation I think it should be Collect Logs for Nucleon

File: README.md [hunk 2]
```diff
@@ -1,26 +1,97 @@
...
+1. Go to SumoLogic home page, and press on ***Setup Wizard***.
+2. Move the cursor to ***Start streaming data to Sumo Logic*** and press on ***Get Started***
+3. Select the following : ***All other Sources***
+4. go to  ***HTTPS Source***
+5. Chose a source category and hit next
+6. copy the URL and send it to Nucleon Cyber (support@cnucleon.sh)
 
+# Dashboard
```
Comment (himanshu219) [inline README.md:37]: Instead of Dashboards it should be Install the Cybereason App and View the Dashboards

Comment (himanshu219) [reply README.md:37]: After the title we can have below lines

```
This page has instructions for installing the Sumo Logic App for Nucleon and descriptions of each of the app dashboards.

Use the instruction from this doc (https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) to install the Nucleon App.
```

And then we can continue with dashboard name and other description.

File: README.md [hunk 3]
```diff
@@ -1,26 +1,97 @@
-# sumologic-public-partner-apps
```
Comment (himanshu219) [inline README.md:1]: We need to update the manifest.json https://github.com/SumoLogic/sumologic-public-partner-apps/blob/master/Nucleon/Nucleon.manifest.json#L9

and update the helpURL field to https://github.com/SumoLogic/sumologic-public-partner-apps/tree/master/Nucleon

**Review by himanshu219 — CHANGES_REQUESTED** [review]

**Review by sumo-apps-team — APPROVED** [review]
Will fix Readme updates in other PR. Merging it for 7th May Release Train.

**Pull request merged**